test/functional/bootstrap-e2e-specs.js

```diff
@@ -0,0 +1,39 @@
+// transpile :mocha
+
+import chai from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+import 'mochawait';
+import path from 'path';
+import AndroidBootstrap from '../../index.js';
```

Comment:
I think that if you import from `../..` it will go through the module code, rather than just the `index.js` file, and so it will test that everything is wired correctly (i.e., the `package.json` is correct, etc.).